src/modules/utils/panel/screens/3dprinter/WatchScreen.cpp

```diff
@@ -250,7 +250,7 @@ void WatchScreen::display_menu_line(uint16_t line)
             break;
         }
         case 1: THEPANEL->lcd->printf("X%4d Y%4d Z%7.2f", (int)round(this->pos[0]), (int)round(this->pos[1]), this->pos[2]); break;
-        case 2: THEPANEL->lcd->printf("%3d%% %2lu:%02lu %3u%% sd", this->current_speed, this->elapsed_time / 60, this->elapsed_time % 60, this->sd_pcnt_played); break;
+		case 2: THEPANEL->lcd->printf("%3d%% %02lu:%02lu:%02lu %3u%% sd", this->current_speed, this->elapsed_time / 3600, (this->elapsed_time % 3600) / 60, this->elapsed_time % 60, this->sd_pcnt_played); break;
```

Comment:
Sorry, I think the white space was a copy/paste error.  I thought I had caught all of them.  On mine, I actually left off the 'sd' at the end and spaced in between.

If you're interested in keeping the formatting, I'm willing to change as necessary.  Otherwise, I'll just let it go.  Please let me know.  Thanks!